src/popup/components/OpenAppsBrowserButton.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <BtnIcon
+    class="apps-browser"
+    data-cy="apps-browser-btn"
+    text="Browser"
+    :icon="GlobeSmallIcon"
+    @click="openAppsBrowserScreen()"
```

Comment:
this change will transform the button into route link, and the icon is not going to display

Reply:
Yes, it will be transformed to routerlink and that's good. Every element that is moving user from one place to another should be a router link.
But I don't get why the icon won't be displayed. The `NotificationsIcon` (bell icon on the header) is a `BtnIcon` rendered as a router link and all is fine.